test/functional/connection/connection.ts

```diff
@@ -81,6 +82,39 @@ describe("Connection", () => {
 
     });
 
+    describe("establishing connection", function() {
+        let connection: Connection;
+        after(() => {
+            if (connection && connection.isConnected)
+                return connection.close();
+
+            return Promise.resolve();
+        });
+
+        it("should throw DriverOptionNotSetError when extra.socketPath is missing", function() {
```

Comment:
probably correct statement is `when both "extra.socketPath" and "host" are missing`